docs/relay/relay.md

```diff
@@ -3,31 +3,51 @@ id: relay
 title: Relay
 ---
 
-Netcode for GameObjects (Netcode) allows you to connect to a host by its IP and port. However, if the host isn't on the same network as you (for example, somewhere over the Internet), you will need some extra services to achieve a successful connection and, as a result, a successful game.
+You can use [Netcode for GameObjects (NGO)](https://docs-multiplayer.unity3d.com/netcode/current/about/) to use an IP address and a port to connect a client to a host. You can use the [Relay Unity Games Service (UGS)](https://docs.unity.com/ugs/en-us/manual/relay/manual/introduction) to connect multiple clients to a host that isn't on the same network.
```

Comment:
I think the official name is "Unity Relay"
```suggestion
You can use [Netcode for GameObjects (NGO)](https://docs-multiplayer.unity3d.com/netcode/current/about/) to use an IP address and a port to connect a client to a host. You can use [Unity Relay](https://docs.unity.com/ugs/en-us/manual/relay/manual/introduction) to connect multiple clients to a host that isn't on the same network.
```
If you want to mention UGS, pay attention that the acronym stands for "Unity Gam**ing** Services"